### ISSUE TITLE
Markers not being rendered

### ISSUE THREAD

**Issue by artemfomin** [lolochristen/OpenLayers.Blazor#42]
When adding a marker to the map using
```
private void OnMapClick(Coordinate coordinate)
{
    _map.MarkersList.Add(new Marker(MarkerType.MarkerFlag, coordinate, "Marker friendly name"));
    StateHasChanged();
}
```
Marker is added to `_map.MarkersList` but isn't rendered.
Also tried
```
private void OnMapClick(Coordinate coordinate)
    {
        _map.MarkersList.Add(new Marker(MarkerType.MarkerFlag, coordinate, DateTime.Now.ToShortTimeString()));
        StateHasChanged();
        _map.SetMarkers(_map.MarkersList);
        _map.UpdateLayer(_mainLayer);
    }
```
Still nothing.

Package version 1.5.3 from 02.02.2024
Package is great and intuitive if only it will be stable)

UPD: Might be related to this [issue](https://github.com/lolochristen/OpenLayers.Blazor/issues/37) as failure of base JS object might block the following calls.

**Comment by lolochristen** [comment]
Please see comment in #37. Hope it is fixed now.

**Comment by artemfomin** [comment]
@lolochristen I confirm both issues are fixed in 1.5.4. Thank you. BLAZing fast)
